.github/workflows/docker_main.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - '**.md'
+      - '.github/**'
```

Comment:
We need the tests to run because we generate the references for the structs through the test

Reply:
The tests currently run whenever a `.go` file changes, as the tests generate the `.md` files and are triggered by `.go` file modifications. In this PR, I propose not running Go tests when only `.md` files are changed since these changes don't impact the Go code.
To summarize, the test `TestCheckGeneratedMarkdownPages` will run when `.go` code changes as expected.

Reply:
Yeah that makes sense but the markdown generated could be incorrect compared to the code that is checked in. So if I go and manually change for some reason the markdown files generated through the tests then the test that checks that the MD generated matches the values in code won't run and the PR checks will be green. Giving the reviewer a false indication that the MD is correct

Reply:
So I will ignore specific paths